src/client/feed/Feed.js

```diff
@@ -2,16 +2,47 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import moment from 'moment';
 
-const Feed = ({ feed }) => (
-  <div>
-    <h3>Search results</h3>
-    <div className="ui divided items">
+const Feed = ({ feed, hiddenWords }) => {
+  const filteredFeed = filterPostsForFeed(feed, hiddenWords);
+
+  return (
+    <div>
+      <h3>Search results, showing {filteredFeed.length} posts.</h3>
+      (Hiding {feed.length - filteredFeed.length} posts)
+      <div className="ui divided items">
+        {
+          filteredFeed
+            .map((feedItem) => (<FeedItem content={feedItem}/>))
+        }
+      </div>
+    </div>
+  );
+}
+
+const filterPostsForFeed = (feed, hiddenWords) => {
+  return feed.filter((feedItem) => {
+    switch (feedItem.source) {
+      case 'twitter':

```

Comment:
I don't know but I feel that's something to do with when we get to it..  

I think we're likely to need to change the switch case anyway because we're unsure of that or what we would use in the future so better not to have code that's not doing anything to confuse us if we look back at it..